script/deploy_gke.sh

```diff
@@ -0,0 +1,56 @@
+#!/bin/bash
+# Copyright 2019 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+
+# Script to deploy mixer to a GKE cluster.
+# This retrives the docker images and gRPC descriptor based on git commit hash,
+# so these binaries should have been pushed to container registry and Cloud
+# Storage.
```

Comment:
Is there a script that needs to be run to do this? Would be good to reference that here.

Reply:
Might be good to also add a warning to do this from a clean repo, and to fetch the latest from upstream (or maybe the script should do that as well?)

Reply:
Also check that the ENV is supplied (and exit script if not specified or invalid). The header comment should include possible options.

Reply:
Updated accordingly